### PR TITLE
handle roleDeletionResponse

### PR DESCRIPTION
**Pull request by udaydenduluri33**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/users/containers/utils/user-roles-util.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import { AppConstants } from '../../../app/app.constants';
 import {propsExist} from '../../../../api/lib/objectUtilities';
```
Comment (paddybasi) [inline src/users/containers/utils/user-roles-util.ts:1]: Can we please try avoiding a cross import between api and frontend? We've had issues with this in the past

File: src/users/containers/user-details/user-details.component.ts [hunk 2]
```diff
@@ -130,6 +130,10 @@ export class UserDetailsComponent implements OnInit, OnDestroy {
     }
   }
 
+  public isSuspendedOrPending(status) {
+    return status === 'Suspended' || status === 'Pending';
```
Comment (paddybasi) [inline src/users/containers/user-details/user-details.component.ts:134]: Maybe make this a bit more 'generic'? ```isInactive``` or something? Could also use an array value for the statuses instead

Comment (udaydenduluri33) [reply src/users/containers/user-details/user-details.component.ts:134]: @paddybasi  this property is set by PRD. Not in our hand

File: src/users/containers/utils/user-roles-util.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/users/containers/utils/user-roles-util.ts:1]: Will do

Comment (udaydenduluri33) [reply src/users/containers/utils/user-roles-util.ts]: Done

File: src/users/containers/user-details/user-details.component.ts (hunk 2 above)
Comment (paddybasi) [reply src/users/containers/user-details/user-details.component.ts:134]: No sorry, I meant the function naming and testing the status value:
```
public isInactive(status, inactiveStatuses: string[] = ['Suspended', 'Pending']) {
   return inactiveStatuses.includes(status);
}
```

Comment (udaydenduluri33) [reply src/users/containers/user-details/user-details.component.ts]: Done

**Pull request merged**